modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -42,6 +42,23 @@ CALL (i) {
 This option applies an exponential delay between retries for transaction batches failing due to transient errors, with an optional maximum retry duration, and handles failure based on a specified fallback error mode if the transaction does not succeed within the given time.
 For more information, see xref:subqueries/subqueries-in-transactions#on-error-retry[`CALL` subqueries in transactions -> `ON ERROR RETRY`].
 
+a|
```

Comment:
did you add this in the `Neo4j 2025.03` section? cause I don't think that'll be the correct one XD

should probably add a section for the next version/the version we think it'll go GA in instead

Reply:
oh, yes. Should be 2025.04 section. Was wrongly assuming that the newest section would be the latest, but I guess nothing else in cypher went into 2025.04 (at least not documented yet)

Reply:
yeah, if you're first out you need to update it :P

also is 2025.04 correct? won't it still be behind a feature flag there and therefore not GA/documented? so it should be in 2025.05 (assuming we remove the feature flag in the next release)?

Reply:
or did we remove the feature flag in the version just cutoff?

Reply:
Yes, it was removed